### PR TITLE
Update `bin/i18n`'s `package-lock.json`

### PR DESCRIPTION
**Pull request by Hamms**
Currently, attempting to install the `bin/i18n` dependencies will produce the following warning:

    [~/code-dot-org/bin/i18n (staging)]$ npm install
    npm WARN old lockfile
    npm WARN old lockfile The package-lock.json file was created with an old version of npm,
    npm WARN old lockfile so supplemental metadata must be fetched from the registry.
    npm WARN old lockfile
    npm WARN old lockfile This is a one-time fix-up, please be patient...
    npm WARN old lockfile

Additionally, encountering this warning while provisioning a new test server from scratch seems to cause some problems with later parts of the build. This PR simply commits the results of applying that fix-up, and with this change we should again be able to provision a new test server without manual intervention.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by sureshc — APPROVED** [review]
I defer to platform team on this, but looks like an important fix.

**Review by cat5inthecradle — APPROVED** [review]
LGTM, if there are any problems caused by this, they're safe and visible.

**Review by wilkie — COMMENTED** [review]
Feels harmless enough... but I didn't try running the sync with it yet. :)

**Pull request merged**
